docs/test/troubleshooting-code-coverage.md

```diff
@@ -51,6 +51,8 @@ The *.pdb* file must be generated from the same build as the *.dll* or *.exe* fi
 
 Resolution&mdash;Make sure that your build settings generate the *.pdb* file. If the *.pdb* files are not updated when the project is built, then open the project properties, select the **Build** page, choose **Advanced**, and inspect **Debug Info**.
 
+For C++ projects, ensure that the generated pdbs have full debug information by opening the project properties and ensuring that **Linker > Debugging > Generate Debug Info** setting is set to **Generate Debug Information optimized for sharing and publishing (/DEBUG:FULL)**.
```

Comment:
```suggestion
For C++ projects, ensure that the generated .pdb files have full debug information. Open the project properties and verify that **Linker** > **Debugging** > **Generate Debug Info** is set to **Generate Debug Information optimized for sharing and publishing (/DEBUG:FULL)**.
```

Also, the "generate debug information..." setting is half capitalized and half not capitalized. Can you copy it exactly as it's written in the product?

Reply:
Thank you @gewarren. I've applied the changes.

Also confirmed that the name of the setting is exactly as it appears in the product -
![image](https://user-images.githubusercontent.com/10579684/65080484-433d0c80-d956-11e9-82e0-4431b48ee20c.png)